spec/std/process_spec.cr

```diff
@@ -181,7 +181,7 @@ describe Process do
     end
 
     it "sets working directory" do
-      parent = File.dirname(Dir.current)
+      parent = Path.new File.dirname(Dir.current)
```

Comment:
Given that `Path` was the cause of the issue, I don't think that there's any need to have specs for it _and_ `String` – the `initialize` method calls `#to_s` so testing for `String` is pointless to me.

Reply:
This would be true if the spec is for `Crystal::System::Process.spawn`. But the paremeter type of `chdir` is _still_ `Path | String?` in `Process`'s API.

Reply:
Part of the point of specs is to define the expected behavior. Just because `Path` becomes `String` now, doesn't mean it would in a possible future refactor. Having both specs adds a safeguard to ensure that future changes to the logic still handles both contexts.